net/libreswan/Makefile

```diff
@@ -7,12 +7,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libreswan
-PKG_VERSION:=3.27
-PKG_RELEASE:=3
+PKG_VERSION:=3.29
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/libreswan/libreswan.git
```

Comment:
This looks like a mistake. Use only one PKG_SOURCE_URL please.

Reply:
Yes, it worked because the other one overwrote it

Reply:
fixed. Thanks @neheb .